Packs/TheHiveProject/Integrations/TheHiveProject/TheHiveProject.py

```diff
@@ -926,6 +926,10 @@ def get_mapping_fields_command(client: Client, args: dict) -> Dict[str, Any]:
 def update_remote_system_command(client: Client, args: dict) -> str:
     parsed_args = UpdateRemoteSystemArgs(args)
     changes = {k: v for k, v in parsed_args.delta.items() if k in parsed_args.data}
+    demisto.debug(f'Changes from update_remote_system: {changes}')
+    # Convert the value to integer as the api request
+    keys_to_convert = ['severity', 'tlp', 'pap']
+    changes = {k: (int(v) if k in keys_to_convert else v) for k, v in changes.items()}
```

Comment:
I understand we basically want to cast digits to integers right? why not use isdigit?
```suggestion
    # Convert the value to integer as the api request=
    changes = {k: (int(v) if v.isdigit() else v) for k, v in changes.items()}
```